UnitTests/ExpectedException/FeatureIntegrationTest.cs

```diff
@@ -0,0 +1,77 @@
+﻿#region copyright
+
+// 
+// Copyright (c) rubicon IT GmbH
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"),
+// to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
+// and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+// 
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+// FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+// WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+#endregion
+
+using System;
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using NUnit.Framework;
+using NUnit2To3SyntaxConverter.ExpectedException;
+
+namespace NUnit2To3SyntaxConverter.UnitTests.ExpectedException
+{
+  public class FeatureIntegrationTest
+  {
+    public (ExpectedExceptionRewriter, MethodDeclarationSyntax) RewriterFor (string file, string methodName)
+    {
+      var (semantic, root) = CompiledSourceFileProvider.LoadSemanticModel (file);
+      var rewriter = new ExpectedExceptionRewriter (semantic, new ExpectedExceptionMethodBodyTransformer(), new ExpectedExceptionAttributeRemover());
+      var method = root.DescendantNodes()
+          .OfType<MethodDeclarationSyntax>()
+          .Single (syntax => syntax.Identifier.ToString() == methodName);
+      return (rewriter, method);
+    }
+
+    [Test]
+    [TestCase (

```

Comment:
For these tests you probably do really need the resource files, but you can still remove clutter by storing them in constants and only passing the method name in the test case